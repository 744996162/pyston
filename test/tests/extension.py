import test

print test
test.store([])
print test.load()

class C(object):
    def __init__(self):
        self.x = self.y = self.z = self.w = self.k = self.a = self.b = self.c = 1

for i in xrange(100000):
    C()

print "This will break"
print test.load()
