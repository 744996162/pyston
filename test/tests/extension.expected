<module 'test' (built-in)>
[]
This will break
[]
